lib/attribute-parser.js

```diff
@@ -152,8 +157,10 @@ module.exports = class AttributeParser {
 
         }
         else {
+
           const exp = 'v = ' + value
-          eval(exp) // eval is evil except in this case!
+          v = vm.runInThisContext(exp)
```

Comment:
weird.. Ok it's working 
i was doing that
````
            const context = new vm.createContext(this.globalContext)
            const script = new vm.Script(exp)
```

The new are useless ?
https://nodejs.org/api/vm.html#vm_script_runincontext_contextifiedsandbox_options